cli/tests/conftest.py

```diff
@@ -198,12 +198,15 @@ def _unset_env_var(key):
 
 @pytest.fixture()
 def run_cli(mocker, capsys):
-    def _run_cli(command, expect_failure=False):
+    def _run_cli(command, expect_failure=False, expect_message=None):
         mocker.patch.object(sys, "argv", command)
         with pytest.raises(SystemExit) as sysexit:
             ParallelClusterCli().handle_command()
         if expect_failure:
-            assert_that(sysexit.value.code).is_greater_than(0)
+            if expect_message:
+                assert_that(sysexit.value.code).contains(expect_message)
+            else:
+                assert_that(sysexit.value.code).is_greater_than(0)
```

Comment:
`sysexit.value.code` is a number or a string based on the arg passed to `sys.exit`? just curious, because i would have thought `.code` is always a number.

Reply:
exactly, the code is a string when passing sys.exit("error message")